src/ComplexSelector.js

```diff
@@ -78,6 +78,13 @@ export default class ComplexSelector {
       return this.handleSelectors(selectors, wrapper);
     }
 
+    if (typeof selector === 'object' && selector !== null) {
+      const hasUndefinedValues = Object.keys(selector).some(key => selector[key] === undefined);
```

Comment:
`import values from 'object.values'` and you can do `values(selector).some(value => value === undefined)`